tests/dragonfly/snapshot_test.py

```diff
@@ -191,3 +191,34 @@ async def test_snapshot(self, df_seeder_factory, df_server):
         await a_client.connection_pool.disconnect()
 
         assert await seeder.compare(start_capture)
+
+@dfly_args({**BASIC_ARGS, "dbfilename": "test-shutdown"})
+class TestDflyInfoPersistenceLoadingField(SnapshotTestBase):
+    """Test is_loading field on INFO PERSISTENCE during snapshot loading"""
+    @pytest.fixture(autouse=True)
+    def setup(self, tmp_dir: Path):
+        self.tmp_dir = tmp_dir
+
+    def extract_is_loading_field(self, res):
+        matcher = b'loading:'
+        start = res.find(matcher)
+        return res[start + len(matcher)]
+
+    @pytest.mark.asyncio
+    async def test_snapshot(self, df_seeder_factory, df_server):
+        seeder = df_seeder_factory.create(port=df_server.port, **SEEDER_ARGS)
+        await seeder.run(target_deviation=0.1)
+
+        df_server.stop()
+        df_server.start()
+
+        a_client = aioredis.Redis(port=df_server.port)
+        res = await a_client.execute_command("INFO PERSISTENCE")
+        assert ord('1') == self.extract_is_loading_field(res)
```

Comment:
I had to think for a second what it actually means 😄 Can't you use the byte prefix like you did above, just extract not a char, but a subslice when you get the field